commitlint.config.ts

```diff
@@ -43,7 +43,12 @@ module.exports = {
         "too-many-spaces": [RuleConfigSeverity.Error, "always"],
         "commit-hash-alone": [RuleConfigSeverity.Error, "always"],
         "title-uppercase": [RuleConfigSeverity.Error, "always"],
+        "proper-revert-message": [RuleConfigSeverity.Error, "always"],
```

Comment:
@realmarv let's rename this rule to `default-revert-message` and change from `always` to `never`; then we should map the always/never argument properly to make `offence` be depending on it

Reply:
Done